### ISSUE TITLE
Double normalization in weighted feature fusion

### ISSUE THREAD

**Issue by oadams** [toandaominh1997/EfficientDet.Pytorch#102]
In `BiFPN.forward()` the feature weights are normalized:

https://github.com/toandaominh1997/EfficientDet.Pytorch/blob/fbe56e58c9a2749520303d2d380427e5f01305ba/models/bifpn.py#L178-L180

In calculating the bottom up and top down features, it appears this normalization is done again unnecessarily:

https://github.com/toandaominh1997/EfficientDet.Pytorch/blob/fbe56e58c9a2749520303d2d380427e5f01305ba/models/bifpn.py#L190

https://github.com/toandaominh1997/EfficientDet.Pytorch/blob/fbe56e58c9a2749520303d2d380427e5f01305ba/models/bifpn.py#L196

**Comment by XiaoyuShi97** [comment]
@oadams Hi guy, I also find this issue and I believe this can cause big problem. Did you try training the model without changing this part?

**Comment by oadams** [comment]
@Shi-Xiaoyu No, I never actually trained the model using this code, I was just reading the code.
